inst/scripts/Shin2020.R

```diff
@@ -0,0 +1,166 @@
+library("MSnbase")
+library("pRoloc")
+
+
+Shin2019MitoControl <- readRDS("../../inst/extdata/Shin2019MitoCon.rds")
```

Comment:
How come you have rds files here? Could you not start from csv files, or whatever initial files you have created these rds files in the first place.